test/CadlRanchProjects.Tests/encode-bytes.cs

```diff
@@ -161,5 +162,44 @@ public class EncodeBytesTests : CadlRanchTestBase
             Response response = await new BytesClient(host, null).GetRequestBodyClient().Base64urlAsync(data);
             Assert.AreEqual(204, response.Status);
         });
+
+        [Test]
+        public Task Encode_Bytes_ResponseBody_default() => Test(async (host) =>
+        {
+            var response = await new BytesClient(host, null).GetResponseBodyClient().DefaultAsync();
+            var expected = BinaryData.FromObjectAsJson("dGVzdA==");
+            Assert.AreEqual(expected.ToString(), response.Value.ToString());
```

Comment:
well let us use the same way to assert binary data, or we could write a private static helper inside this class such as `AssertBinaryDataAreEqual` to assert them

Reply:
fixed